config/merge/merge.go

```diff
@@ -213,6 +213,7 @@ func MergeStructTranscribe(parent, child interface{}) (interface{}, Transcript)
 // parent and child MUST be the same type
 // we transcribe all leaf fields, and all intermediate structs that wholly
 // originate from either parent or child
+// nolint:staticcheck
```

Comment:
Let's put this on the actual affected line, if possible, rather than the entire function.